### DISCUSSION TITLE
How to get a QR Code for my Linkfree Profile?

### DISCUSSION THREAD

**Discussion by himanshubalani** [EddieHubCommunity/LinkFree#4403] [Q&A] [answered]
I saw that QR Code for Profiles was a feature on LinkFree Homepage. However I can't find the Documentation on generating a QR Code or where to find it. Can someone help me out?

**Comment by himanshubalani** [comment, marked as answer]
Oh okay found It at my profile image. I am sorry for spam

**Reply by eddiejaoude** [reply]
no problem, glad you found it - any suggestions to help others about the QR code let us know

**Comment by himanshubalani** [comment]
Thank you very much Eddie! I think the cause  could have been that-
1.It wasn't apparent for me that the QR code was present there. I thought it was just an animated design first.
2. The QR code image inside the button is small(to my eyes atleast).

**Reply by eddiejaoude** [reply]
thank you for the feedback 👍

**Reply by himanshubalani** [reply]
@eddiejaoude Thank you very much for taking my feedback. I saw the change on my profile today and it's really admirable of you to fix this.  The button is much more prominent now and easily distinguishable.
